austation/code/game/gamemodes/bloodsucker/bloodsucker.dm

```diff
@@ -0,0 +1,296 @@
+
+/datum/game_mode
+	var/list/datum/mind/bloodsuckers = list() 		// List of minds belonging to this game mode.
+	var/list/datum/mind/vassals = list() 			// List of minds that have been turned into Vassals.
+	var/list/datum/mind/vamphunters = list() 		// List of minds hunting vampires.
+	var/obj/effect/sunlight/bloodsucker_sunlight	// Sunlight Timer. Created on first Bloodsucker assign. Destroyed on last removed Bloodsucker.
+
+	// LISTS //
+	var/list/vassal_allowed_antags = list(/datum/antagonist/brother, /datum/antagonist/traitor, /datum/antagonist/traitor/internal_affairs, /datum/antagonist/survivalist, \
+										  /datum/antagonist/rev, /datum/antagonist/nukeop, /datum/antagonist/pirate, /datum/antagonist/cult, /datum/antagonist/abductee)
+	// The antags you're allowed to be if turning Vassal.
+
+/datum/game_mode/bloodsucker
+	name = "bloodsucker"
+	config_tag = "bloodsucker"
+	report_type = "traitor"
+	traitor_name = "Bloodsucker"//Nanotrasen Internal Affairs Agent"
+	antag_flag = ROLE_BLOODSUCKER
+	false_report_weight = 1
+	restricted_jobs = list("AI","Cyborg")
+	protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Head of Personnel", "Deputy")
+	required_players = 0
```

Comment:
So this is spicy. Terra please look at this.

Reply:
Yep, that was swain. He wanted bloodsuckers to always be a gamemode to do, Terra will maybe want to change that to like 12 or 15 or something. As far as I looked at the code, there will be a min of 1 to max of 6 bloodsuckers. For every 10 players a new bloodsucker would be made, so minimum 10 players would make the gamemode balanced from how Swain coded it.

Reply:
Uhh yeah. From what I’ve read 10 would be a bit of a stretch too. May I suggest you set it to 20-25? We usually have highpop now so you don’t need to worry about it not getting seen much :)